safe/utilities/expressions.py

```diff
@@ -0,0 +1,33 @@
+# coding=utf-8
+
+"""Utilities module related to QGIS Expressions."""
+
+from inspect import getmembers
+
+
+from safe.gis import generic_expressions
+from safe.report.expressions import infographic, map_report
+
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+def qgis_expressions():
```

Comment:
Does this duplicate code from definitions_help? Maybe we can refactor that if you did not do so already....

Reply:
It's done already. Only one place to retrieve all QGIS expressions